packages/core-js/internals/inspect-source.js

```diff
@@ -8,7 +8,12 @@ var functionToString = uncurryThis(Function.toString);
 // this helper broken in `core-js@3.4.1-3.4.4`, so we can't use `shared` helper
 if (!isCallable(store.inspectSource)) {
   store.inspectSource = function (it) {
-    return functionToString(it);
+    try {
+      return isCallable(it) ? functionToString(it) : '';
```

Comment:
It should throw a error for non-callable objects, as I wrote take a look at https://github.com/zloirock/core-js/issues/1381#issuecomment-2500857379